ci-operator/step-registry/workers-rhel/vsphere-provision/workers-rhel-vsphere-provision-commands.sh

```diff
@@ -47,27 +47,43 @@ for count in $(seq 1 ${RHEL_WORKER_COUNT}); do
     exit 1
   fi
 
-  echo ${rhel_node_ip} >> /tmp/rhel_nodes_ip
+  echo "${rhel_node_ip}" >> /tmp/rhel_nodes_ip
```

Comment:
Can we directly output them to `${SHARED_DIR}`

Reply:
`/tmp/rhel_nodes_ip` is only used in current ref, and write into ansible hosts file later, so looks like no need to output to `${SHARED_DIR}`

Reply:
If I remember right, `${SHARED_DIR}` is pointed to `/tmp` too, right?

Reply:
seems yes, I noticed shared files are under /tmp folder. But in this case, file` /tmp/rhel_nodes_ip` is no need to shared with other steps, so directly created in `/tmp` folder.

Reply:
So sounds like `/tmp/rhel_nodes_ip` == `${SHARED_DIR}/rhel_nodes_ip`.

Another reason I am asking here is because I saw the following 61 line will copy `/tmp/rhel_nodes_info` to `SHARED_DIR`. So I am thinking we can directly put them in `${SHARED_DIR}` to save some codes.

Reply:
And I am thinking if we can drop `/tmp/rhel_nodes_ip`, directly use `${SHARED_DIR}/rhel_nodes_info`. And put them in `${ARTIFACT_DIR}` is neccessary?